qcfractal/interface/client.py

```diff
@@ -79,6 +80,18 @@ def __init__(self,
 
         self.server_name = self.server_info["name"]
 
+        def version_list(version):
+            re.sub(r"\+.*$", "", version.strip("v"))
+            return [int(x) for x in version.split(".")]
+
+        from . import __version__  # Import here to avoid circular import from __init__
+        server_version = version_list(self.server_info["version"])
+        client_version = version_list(__version__)
+        assert client_version[:-1] >= server_version[:-1], (f"Sever's' Fractal version of {server_version} appears to "
```

Comment:
I think for now major/minor must match. Backwards compute is not guaranteed either.

Reply:
So right now the server is on 0.5.4 but the developmental stuff we are doing is on 0.6.0, if we make them match, we will stop being able to develop or run production at this moment unless we downgrade. 

I also don't think this PR should be merged in until we have a set upgrade path/procedure in place for the server.